### PR TITLE
refactor(react-chart): remove extra data fields

### PR DESCRIPTION
**Pull request by DmitryBogomolov**

### CODE REVIEW

**Comment by Krijovnick** [comment]
Title of the PR is not correct. Change it pls

**Comment by DmitryBogomolov** [comment]
Done.

**Comment by Krijovnick** [comment]
Streamgraph is wrong.

Before:
image

After:
image

**Comment by Krijovnick** [comment]
Chart with negative values rendered incorrectly
image

File: packages/dx-react-chart/src/utils/series-helper.jsx [hunk 1]
```diff
@@ -91,12 +102,14 @@ export const withSeriesPlugin = (
   }
   Component.propTypes = {
     name: PropTypes.string,
+    /* eslint-disable react/no-unused-prop-types */
```
[the remark this answers is not in the document]
Comment (DmitryBogomolov) [reply packages/dx-react-chart/src/utils/series-helper.jsx]: The group of props can be removed (presence of `argumentField` and `valueField` would not be checked in *dev* mode then). If we want to preserve the check the *eslint* comments are required.

**Pull request merged**